homeassistant/components/habitica/config_flow.py

```diff
@@ -79,6 +80,19 @@ async def async_step_user(self, user_input=None):
 
     async def async_step_import(self, import_data):
         """Import habitica config from configuration.yaml."""
+        async_create_issue(
+            self.hass,
+            HOMEASSISTANT_DOMAIN,
+            f"deprecated_yaml_{DOMAIN}",
+            is_fixable=False,
+            breaks_in_ha_version="2024.12.0",
+            severity=IssueSeverity.WARNING,
+            translation_key="deprecated_yaml",
+            translation_placeholders={
+                "domain": DOMAIN,
+                "integration_title": "Habitica",
+            },
+        )
```

Comment:
Let's leave this one for a separate PR btw

Reply:
Because that counts as deprecation, while the rest of the PR is a new feature

Reply:
ok 👍